app/constants/index.ts

```diff
@@ -26,7 +26,7 @@ export const STATUS_PAGE_URL = 'http://status.test-blockstack.com';
 
 export const DEFAULT_STACKS_NODE_URL = whenNetwork<string>({
   mainnet: 'https://stacks-node-api.mainnet.stacks.co',
-  testnet: 'https://stacks-node-api.testnet.stacks.co',
+  testnet: 'https://stacks-node-api-microblocks.testnet.stacks.co',
```

Comment:
Not to be included in release, for testing I was adding this manually

Reply:
Yes this is just so that QA can download the build and test. I will remove it after.

Reply:
reverted this